spec/support/shared_contexts/desperado.rb

```diff
@@ -0,0 +1,24 @@
+require "rake"
+
+##
+# Define a shared context that loads our "desperado" tasks.
+#
+# {Link Test Rake Tasks Like a BOSS}[https://robots.thoughtbot.com/test-rake-tasks-like-a-boss]
+#
+shared_context 'desperado' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.